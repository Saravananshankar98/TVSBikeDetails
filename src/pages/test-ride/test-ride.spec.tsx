import { render } from "@testing-library/react";
import TestRide from "./test-ride";

describe("DashBoard", () => {
  let baseElement: any, container: any, screen;
  beforeEach(() => {
    ({ container, baseElement, ...screen } = render(<TestRide />));
  });

  it("Matches Snapshot", () => {
    expect(baseElement).toMatchSnapshot();
  });
});
