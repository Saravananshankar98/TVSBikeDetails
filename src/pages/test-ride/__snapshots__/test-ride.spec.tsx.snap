// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashBoard Matches Snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root css-0"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-grid-sm-12 css-139gbf4-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-grid-xs-6 css-1jssz31-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-0"
          >
            <img
              alt="BookVehicles"
              src="https://www.tvsmotor.com/-/media/Feature/Book-A-Ride/bs-vi-range.jpg"
            />
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-grid-xs-6 css-1jssz31-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              <h3>
                 Book A Test Ride
              </h3>
               
              <br />
              To book your test ride now, kindly fill in your information below.
            </p>
            <div
              class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard css-aqpgxn-MuiFormLabel-root-MuiInputLabel-root"
                data-shrink="false"
                for="name"
                id="name-label"
              >
                Name*
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl css-v4u5dn-MuiInputBase-root-MuiInput-root"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                  id="name"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
